tests/llm/utils/test_case_utils.py

```
@@ -34,7 +34,18 @@ def __init__(
     models_str = os.environ.get("MODEL", "gpt-4o")
-    return models_str.split(",")
+    models = models_str.split(",")
+
+    # If multiple models are specified, require explicit CLASSIFIER_MODEL
+    if len(models) > 1 and not os.environ.get("CLASSIFIER_MODEL"):
+        raise ValueError(
+            f"Multiple models specified ({models_str}) but CLASSIFIER_MODEL not set. "
+            "When testing multiple models, you must explicitly set CLASSIFIER_MODEL "
+            "to ensure consistent scoring. Example:\n"
+            "  MODEL=gpt-4o,claude-3-5-sonnet CLASSIFIER_MODEL=gpt-4o poetry run pytest tests/llm/"
+        )
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)